steps/set-terraform-varfile-env-var.yaml

```diff
@@ -0,0 +1,30 @@
+parameters:
+  terraformVarFile: ''
+  environment: ''
+
+steps:
+  - pwsh: |
+      $terraformVarFile = $env:terraformVarFile
```

Comment:
I think this could be far simpler in bash with just

```
if [ -f $FILE_PATH ] ;then
  set output...
fi
``

Reply:
yaml step that has oowershell is removed, in favour of having logic in yaml template